### DISCUSSION TITLE
When is the next release date?

### DISCUSSION THREAD

**Discussion by arwedus** [useblocks/sphinx-needs#474] [Q&A] [answered]
@danwos I am currently running on head because I need a bugfix that is not yet released, but would like to change my package config to a new minor version again (i.e. 0.7.5).

When do you plan to create the next patch release?

**Comment by danwos** [comment]
Hi @arwedus,

I'm planing it this week.
I just set up a release pipeline for our CI, so that release creation and publishing is much easier in future.

There are still 1-2 PRs open , which I want to have inside the upcoming release. But I think its doable.

**Comment by danwos** [comment, marked as answer]
It's out: https://pypi.org/project/sphinxcontrib-needs/0.7.5/
